go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/tursodatabase/libsql-client-go
 go 1.20
 
 require (
-	github.com/antlr/antlr4/runtime/Go/antlr/v4 v4.0.0-20230512164433-5d1fd1a340c9
-	github.com/libsql/sqlite-antlr4-parser v0.0.0-20230802215326-5cb5bb604475
+	github.com/antlr4-go/antlr/v4 v4.13.0
```

Comment:
This is the correct Go module to pull in

https://pkg.go.dev/github.com/antlr4-go/antlr/v4